README.md

```diff
@@ -1,2 +1,23 @@
 # hw01
 Repository for homework 1
+
+## Introduction
```

Comment:
Hi Clara, sorry for this late peer-review, I misunderstood the instructions on the syllabus. I'm peer-reviewing your assignment from this week as I was not assigned to anyone.